Docs/source/install/hpc/karolina.rst

```diff
@@ -0,0 +1,197 @@
+         cmake -S . -B build_gpu -DWarpX_COMPUTE=CUDA -DWarpX_PSATD=ON -DWarpX_LIB=ON -DWarpX_DIMS="1;2;RZ;3"
+         cmake --build build_gpu -j 16
+         cmake --build build_gpu -j 16 --target pip_install
+
+      **That's it!**
+      The WarpX application executables are now in ``$HOME/src/warpx/build_gpu/bin/`` and we installed the ``pywarpx`` Python module.
+
+   .. tab-item:: CPU Nodes
+
+      CPU usage is documentation is TODO.
+
+Now, you can :ref:`submit Karolina compute jobs <running-cpp-karolina>` for WarpX :ref:`Python (PICMI) scripts <usage-picmi>` (:ref:`example scripts <usage-examples>`).
+Or, you can use the WarpX executables to submit Karolina jobs (:ref:`example inputs <usage-examples>`).
+For executables, you can reference their location in your :ref:`job script <running-cpp-karolina>` or copy them to a location in ``$PSCRATCH``.
+
+
+.. _building-karolina-update:
+
+Update WarpX & Dependencies
+---------------------------
+
+If you already installed WarpX in the past and want to update it, start by getting the latest source code:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+
+   # read the output of this command - does it look ok?
+   git status
+
+   # get the latest WarpX source code
+   git fetch
+   git pull
+
+   # read the output of these commands - do they look ok?
+   git status
+   git log # press q to exit
+
+And, if needed,
+
+- :ref:`update the karolina_gpu_warpx.profile or karolina_cpu_warpx.profile files <building-karolina-preparation>`,
+- log out and into the system, activate the now updated environment profile as usual,
+- :ref:`execute the dependency install scripts <building-karolina-preparation>`.
+
+As a last step, clean the build directory ``rm -rf $HOME/src/warpx/build_*`` and rebuild WarpX.
+
+
+.. _running-cpp-karolina:
+
+Running
+-------
+
+.. tab-set::
+
+   .. tab-item:: A100 (40GB) GPUs
+
+      The batch script below can be used to run a WarpX simulation on multiple nodes (change ``-N`` accordingly) on the supercomputer Karolina at IT4I.
+      This partition as up to `72 nodes <https://docs.it4i.cz/karolina/hardware-overview/>`__ with 8x A100 (40GB) GPUs each.
+
+      Replace descriptions between chevrons ``<>`` by relevant values, for instance ``<proj>`` could be ``plasma``.
+      Note that we run one MPI rank per GPU.
+
+      .. literalinclude:: ../../../../Tools/machines/karolina-it4i/karolina_gpu.qsub
+         :language: bash
+         :caption: You can copy this file from ``$HOME/src/warpx/Tools/machines/karolina-it4i/karolina_gpu.qsub``.
+
+      To run a simulation, copy the lines above to a file ``karolina_gpu.qsub`` and run
+
+      .. code-block:: bash
+
+         qsub karolina_gpu.qsub
+
+      to submit the job.
+
+
+   .. tab-item:: CPU Nodes
+
+      CPU usage is documentation is TODO.
+
+
+.. _post-processing-karolina:
+
+Post-Processing
+---------------
+
+.. note::
+
+   This section was not yet written.
+   Usually, we document here how to use a Jupyter service.
```

Comment:
After consulting the online documentation and contacting IT4i support, it seems Karolina does not offer a Jupyter service.